### PR TITLE
ci: 🎡 activate gh action to check terraform module versions

### PR DESCRIPTION
**Pull request by jaskaransarkaria**

### CODE REVIEW

File: .github/workflows/check-terraform-module-version.yaml [hunk 1]
```diff
@@ -8,8 +8,8 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - name: Check for editted files for terraform modules
-        uses: ministryofjustice/cloud-platform-environments/cmd/check-terraform-modules-are-latest@add-gh-action-to-pin-terraform-modules-to-latest 
+      - name: Check editted files for terraform modules
```
Comment (jasonBirchall) [inline .github/workflows/check-terraform-module-version.yaml:11]: Small change: `Check edited files for terraform modules`

File: .github/workflows/check-terraform-module-version.yaml [hunk 2]
```diff
@@ -8,8 +8,8 @@ jobs:
...
+        uses: ministryofjustice/cloud-platform-environments/cmd/check-terraform-modules-are-latest@main
```
Comment (jasonBirchall) [inline .github/workflows/check-terraform-module-version.yaml:12]: Do we want to pin the version?

File: .github/workflows/check-terraform-module-version.yaml (hunk 1 above)
Comment (jaskaransarkaria) [reply .github/workflows/check-terraform-module-version.yaml:11]: Thanks!

File: .github/workflows/check-terraform-module-version.yaml (hunk 2 above)
Comment (jaskaransarkaria) [reply .github/workflows/check-terraform-module-version.yaml:12]: Given that it is only our team using the action and currently we are only using it within this repository at the minute, I think it's ok to have it point at 'main'

File: .github/workflows/check-terraform-module-version.yaml [hunk 3]
```diff
@@ -8,11 +8,11 @@ jobs:
...
       - uses: actions/checkout@v3
 
-      - name: Check for editted files for terraform modules
-        uses: ministryofjustice/cloud-platform-environments/cmd/check-terraform-modules-are-latest@add-gh-action-to-pin-terraform-modules-to-latest 
+      - name: Check edited files for terraform modules
+        uses: ministryofjustice/cloud-platform-environments/cmd/check-terraform-modules-are-latest@main
         env:
           REPO_NAME: cloud-platform-environments
-          API_URL: ${{ env.TERRAFORM_MODULE_VERSIONS_API_URL }}
+          API_URL: ${{ vars.TERRAFORM_MODULE_VERSIONS_API_URL }}
```
Comment (jaskaransarkaria) [inline .github/workflows/check-terraform-module-version.yaml:15]: Previously, we couldn't access the env var as we were using the wrong context, to access configuration variables we need to use the vars context.

**Review by jasonBirchall — APPROVED** [review]

**Pull request merged**